packages/styles/scss/components/masthead/_masthead.scss

```diff
@@ -305,6 +315,68 @@ $search-transition-timing: 95ms;
     }
   }
 
+  .#{$prefix}--header__action-left-caret,
+  .#{$prefix}--header__action-right-caret {
+    width: 40px;
+    display: none;
+    position: relative;
+    @media (min-width: 800px) {
+      display: block;
+    }
+  }
+  .#{$prefix}--header__action-left-caret {
+    &::before {
+      content: '';
+      display: block;
+      position: absolute;
+      left: -2px;
```

Comment:
Is this number of centering the caret icon, too? If so, do we want to apply `translateX()` trick here, too, like we do for another dimension?